src/functions/getPassCertificateMissingCertificates/framework/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { APIGatewayProxyEvent, Context } from 'aws-lambda';
+import Response from '../../../common/application/api/Response';
+import createResponse from '../../../common/application/utils/createResponse';
+import { HttpStatus } from '../../../common/application/api/HttpStatus';
+import { bootstrapConfig } from '../../../common/framework/config/config';
+import { bootstrapLogging, error } from '@dvsa/mes-microservice-common/application/utils/logger';
+import { identifyCertificates } from './repository/missing-certificate-repository';
+import { Books, Certificates } from '../../../common/domain/certificates';
+import {
+  findMissingCerts,
+} from '../application/certificate-service';
+
+export async function handler(event: APIGatewayProxyEvent, fnCtx: Context): Promise<Response> {
+
+  try {
+    bootstrapLogging('identify missing certificates', event);
```

Comment:
We should probably standardise this name, current format i've been using is `kebab-case`.

After the comment is addressed here https://github.com/dvsa/mes-result-service/pull/297#issuecomment-1635493926, this would then become 

`bootstrapLogging('get-missing-pass-certificates', event);`